packages/console/src/containers/ConsoleContent/index.tsx

```diff
@@ -15,8 +15,32 @@ import { Skeleton } from './Sidebar';
 import useTenantScopeListener from './hooks';
 import styles from './index.module.scss';
 
+const suspenseDisplayTimeout = 500; // Milliseconds
 const Sidebar = safeLazy(async () => import('./Sidebar'));
 
+function SuspenseFallback() {
+  const timerRef = useRef<number>();
+  const [showSpinner, setShowSpinner] = useState(false);
+
+  useEffect(() => {
+    // eslint-disable-next-line @silverhand/fp/no-mutation
+    timerRef.current = window.setTimeout(() => {
```

Comment:
do we need a ref or just a `const` inside `useEffect`?

Reply:
Ah, you're right. SInce we don't change the timeout, there's no need for the ref.